src/platform/packages/shared/kbn-lens-embeddable-utils/config_builder/schema/charts/metric.ts

```diff
@@ -171,6 +193,7 @@ const metricStateBreakdownByOptionsSchema = schema.object({
 });
 
 export const metricStateSchemaNoESQL = schema.object({
+  id: schema.maybe(schema.string({ meta: { description: 'The chart identifier.' } })),
```

Comment:
is the id part of the schema here? if so, why not adding also to `esqlMetricState`?

Reply:
how is this change related ?